lib/msf/core/auxiliary/scanner.rb

```diff
@@ -63,7 +63,7 @@ def run
   @range_percent = 0
 
   threads_max = datastore['THREADS'].to_i
-  @tl = []
+  @stl = []
```

Comment:
What are your thoughts on renaming this variable to be more descriptive

```suggestion
  @thread_list = []
```

Reply:
Works for me, updated!